tests/test_js_graph.py

```diff
@@ -660,6 +660,26 @@ def test_remove_cells(graph_cases) -> None:
     assert graph.get_edge(*vertices) is None
 
 
+def test_clone_cells(graph_cases) -> None:
+    """
+    :type graph_cases: qmxgraph.tests.conftest.GraphCaseFactory
+    """
+    graph = graph_cases('2v_1e')
+
+    vertices = graph.get_vertices()
```

Comment:
I also often do this:
```suggestion
    vert1, vert2 = graph.get_vertices()
```